### DISCUSSION TITLE
Recoll / database query integration

### DISCUSSION THREAD

**Discussion by make-github-pseudonymous-again** [davatorium/rofi#1853] [Q&A]
Is there a way to use `rofi` as the UI for an update-as-you-type database search? There is script mode, but it seems it is only intended to react to item selection or some custom key strokes. Also there does not seem to be a way to disable the default filtering.

I guess it would make more sense to write a small UI widget from scratch?

**Comment by DaveDavenport** [comment]
maybe rofi-blocks would work for you? https://github.com/OmarCastro/rofi-blocks
Or write your own C plugin for rofi?

**Reply by make-github-pseudonymous-again** [reply]
OK. Both sound reasonable. Any idea if they can be combined with `-dmenu`?

**Reply by DaveDavenport** [reply]
no, that conceptually cannot work.

**Reply by make-github-pseudonymous-again** [reply]
I think I meant if they can be combined with `-theme dmenu`. And now I think I can answer that question myself. Sorry for the confusion. Certainly makes not sense with `-dmenu` I agree.

**Reply by DaveDavenport** [reply]
Yes, themes are independent of the mode used. So it should work with any theme.
